warehouse/accounts/views.py

```diff
@@ -793,7 +794,18 @@ def _error(message):
     if not last_login.tzinfo:
         last_login = pytz.UTC.localize(last_login)
     if user.last_login > last_login:
-        # TODO: track and audit this, seems alertable
+        sentry_sdk.set_context(
+            "user",
+            {
+                "username": user.username,
+                "last_login": user.last_login,
+                "token_last_login": last_login,
+            },
+        )
+        sentry_sdk.capture_message(
+            "Password reset token used after user logged in",
+            level="warning",
```

Comment:
Should this include a fingerprint or is the context sufficient to report different messages per user (assuming we want that).

Reply:
What kind of fingerprint are you thinking of?

Reply:
Like this: https://github.com/pypi/warehouse/blob/e7e9d4ccadc852d33b7c475eaa2f749086227a09/warehouse/oidc/models/_core.py#L178-L182

Reply:
Maybe unique on the username?

Reply:
Oh, I see! I'll update the message to include username in it, and that ought to trigger an issue per username. No need to much with custom scope or fingerprinting for that.